app/config.py

```diff
@@ -369,7 +369,7 @@ class Development(Config):
     TRANSIENT_UPLOADED_LETTERS = 'development-transient-uploaded-letters'
     LETTER_SANITISE_BUCKET_NAME = 'development-letters-sanitise'
 
-    ADMIN_CLIENT_SECRET = 'dev-notify-secret-key'
+    API_INTERNAL_SECRETS = json.loads(os.environ.get('API_INTERNAL_SECRETS', '[]'))
```

Comment:
If we aren't going to have a default here can we add a comment in the README (in the section we talk about environment variables), please?

Reply:
Sorry, that was an accidental commit. I've put it back to the default that it should have been. This should fix the concourse tests too.